deploy/backup/backup.yaml

```diff
@@ -1,11 +1,10 @@
 apiVersion: psmdb.percona.com/v1
 kind: PerconaServerMongoDBBackup
 metadata:
-  finalizers:
-  - delete-backup
```

Comment:
Why do you remove the finalizer?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1112/commits/e8f6ce8911e7c1deb5c4e53815b85a2f56da7c98